platform/linux-generic/pktio/loop.c

```diff
@@ -155,7 +155,7 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 
 		/* Try IPsec inline processing */
 		if (pktio_entry->s.config.inbound_ipsec &&
-		    odp_packet_has_ipsec(pkt))
+		    pkt_hdr->p.input_flags.ipsec)
```

Comment:
Better to use an inline form `_odp_packet_has_ipsec()`, following the model of recent inline enhancements posted by @psavol ?